src/components/KSlideout/KSlideout.vue

```diff
@@ -117,178 +97,159 @@ const emit = defineEmits<{
     overflow-y: auto;
+    padding: var(--kui-space-70, $kui-space-70) var(--kui-space-0, $kui-space-0) var(--kui-space-0, $kui-space-0) var(--kui-space-70, $kui-space-70);
     position: fixed;
     right: 0;
     top: v-bind('offsetTopValue');
     width: 100%;
     z-index: 9999;
 
-    .close-button-start {
-      align-self: flex-start;
-      background: none;
-      border: none;
-      cursor: pointer;
+    .slideout-header {
       display: flex;
-      height: auto;
-      margin-left: var(--kui-space-50, $kui-space-50);
-      margin-top: var(--kui-space-50, $kui-space-50);
-      outline: inherit;
-      position: absolute;
-      transition: $tmp-animation-timing-2 ease;
-
-      &:focus{
-        box-shadow: 0 0 0 2px var(--kui-color-border-primary, $kui-color-border-primary);
+      justify-content: space-between;
+      padding-right: var(--kui-space-70, $kui-space-70);
+
+      .slideout-title {
+        display: flex;
+        flex: 1;
+        font-family: var(--kui-font-family-text, $kui-font-family-text);
+        font-size: var(--kui-font-size-60, $kui-font-size-60);
+        font-weight: var(--kui-font-weight-bold, $kui-font-weight-bold);
+        gap: var(--kui-space-40, $kui-space-40);
+        letter-spacing: var(--kui-letter-spacing-minus-40, $kui-letter-spacing-minus-40);
+        line-height: var(--kui-line-height-50, $kui-line-height-50);
+        user-select: none;
+
+        :deep(#{$kongponentsKongIconSelector}) {
+          color: var(--kui-color-text-neutral, $kui-color-text-neutral) !important;
+        }
       }
-    }
 
-    .close-button-end {
-      align-self: flex-end;
-      background: none;
-      border: none;
-      cursor: pointer;
-      display: flex;
-      height: auto;
-      margin-right: var(--kui-space-50, $kui-space-50);
-      margin-top: var(--kui-space-50, $kui-space-50);
-      outline: inherit;
-      position: absolute;
-      transition: $tmp-animation-timing-2 ease;
+      .slideout-close-icon {
+        border-radius: var(--kui-border-radius-20, $kui-border-radius-20);
+        cursor: pointer;
+        margin-left: auto;
+        // margin-top: var(--kui-space-10, $kui-space-10);
+        outline: none;
 
-      &:focus{
-        box-shadow: 0 0 0 2px var(--kui-color-border-primary, $kui-color-border-primary);
-      }
-    }
+        &:hover, &:focus {
+          color: var(--kui-color-text-neutral-strong, $kui-color-text-neutral-strong) !important;
+        }
 
-    .content {
-      height: 100%;
-      -ms-overflow-style: none;  // IE 10+
-      scrollbar-width: none;  // Firefox
-      &::-webkit-scrollbar {
-        display: none;
+        &:focus-visible {
+          box-shadow: var(--kui-shadow-focus, $kui-shadow-focus);
+        }
       }
+    }
 
-      .content-card {
-        border: none;
-      }
+    .slideout-content {
+      box-sizing: border-box;
+      color: var(--kui-color-text, $kui-color-text);
+      display: flex;
+      flex-direction: column;
```

Comment:
**Issue**: In our application, I find that the slideout content can now trigger a vertical scrollbar despite there being plenty of space.

| Before | After |
|:------:|:-----:|
| ![image](https://github.com/Kong/kongponents/assets/5774638/006ff9d5-7412-4f7b-b6f2-c7b3124cbcda) | ![image](https://github.com/Kong/kongponents/assets/5774638/3e13a8ad-e27f-461b-8a2c-90955eac2c1e) |

There is probably a more elegant solution for this, but in the meantime, I found `flex-grow: 1` to work:


```suggestion
      flex-direction: column;
      flex-grow: 1;
```

Reply:
Addressed this issue here: https://github.com/Kong/kongponents/pull/2025

Reply:
Updated now, please pick up latest version of preview package

Reply:
All good now, thanks 👍🏻